director/interfaces.go

```diff
@@ -95,7 +94,10 @@ type StemcellArchive interface {
 //go:generate counterfeiter . FileReporter
 
 type FileReporter interface {
-	TrackUpload(int64, io.ReadCloser) ui.ReadSeekCloser
```

Comment:
let's make type ReadSeekCloser interface in director package. too ugly with so many same anonymous interfaces.

Reply:
Done